archive/generic-core-terms.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Security is core to our values, and we value the input of hackers acting in good-faith to help us maintain a high standard for the security and privacy for our users. This includes encouraging responsible vulnerability research and disclosure. This policy sets out our definition of good-faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
+
+# Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Work with you to understand and validate your report, including a timely initial response to the submission;
+- Work to remediate discovered vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to report a unique vulnerability, and your report triggers a code or configuration change.
+
+# Scope
+
+# Out-of-Scope
+
+# Rewards
+
+# Disclosure Policy
+
+# Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between legitimate research and malicious attack, we ask that you attempt, in good faith, to:
```

Comment:
Rephrase "malicious attack" with something akin to "unauthorized access" ? "illegal activities" ? "unauthorized activities" ? "illegitimate activities" ?

Reply:
I'm a fan of "illegal activities" as a replacement here.

Reply:
Malicious attacks are almost always illegal and not legitimate research. But legitimate research can still sometimes be an illegal activity. It's splitting hairs to some extent, but my vote is to stick with "malicious attack."